src/core/SystemColor.cpp

```diff
@@ -3,38 +3,34 @@
 
 #ifndef Q_OS_MAC
 
-#define systemColor(dark, light) \
-	bool darkTheme = Config::value("dark_theme", false).toBool(); \
-	return darkTheme ? dark : light;
-
 QColor SystemColor::red()
 {
-	systemColor(qRgb(0xF0, 0x90, 0x90), qRgb(0xb0, 0x18, 0x18))
+	return Config::inDarkMode() ? QColor(0xF0, 0x90, 0x90) : QColor(0xb0, 0x18, 0x18);
```

Comment:
Why not keep the macro?

Reply:
why use a macro when you can use a static method that works everywhere ?

Reply:
To prevent modify 7 lines like you do when you only need to change how dark and light is selected

A macro will not change the usage of the static method. You only avoid to repeat the same logic several times